CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 2.0.0 (Unreleased)
 ==================
+- [BREAKING] Removed make_result from query.py and view.py. Replaced instances of make_result with a Result or QueryResult object.
```

Comment:
This is confusing.  Information about the module is not as important as is the Class that the method if being removed from.  Perhaps change this to just one sentence:  `Removed make_result method from View and Query classes`.